PolicyDecisionPoint/src/test/java/it/cnr/iit/ucs/pdptest/PDPTest.java

```diff
@@ -89,7 +91,7 @@ public String getName() {
     }
 
     @Test
-    public void testPDP() {
+    public void testPDP() throws PolicyException, RequestException {

```

Comment:
I think test methods should not throw exceptions, but instead they should catch it and then fail the test
e.g.
try{
    ....
   assert()
} catch(Exception e){
   fail("should have completed OK");
}

Reply:
ok I've changed this